lessons/module-3/unit-testing-react.md

```diff
@@ -35,11 +35,10 @@ module: 3
 **Golden Rule**: No copy and pasting. Part of the point of this exercise is to
 build up some muscle memory.
 
-When testing React apps, or really when testing any kind of application, most of
-the tests your write will be *unit tests*. Unit tests make sure this small,
-isolated, parts (units) of our application behave as expected. The more we can
-write our code in a way that is testable at the unit level, the easier it will
-be to scale and extend our applications.
+When testing any kind of application, most of
+the tests your write will be *unit tests*.  For practical purposes a unit is a
```

Comment:
"For practical purposes a unit is a function" <- This is not necessarily true, and is a bit misleading. I don't think this section needs changes (aside from a typo I spotted in there)

Reply:
I have revised again - in agreement with your comment.  One pattern you may recognize in a lot of my changes is toward a slightly less chatty voice.  Also, trying to stay in the present tense.  From my perspective of the freshed-eyed learner, some of the chattiness is a distraction from the content.  But becoming too formal is a potential barrier to people trying to read and learn, too.